src/41_secw-rest.xml

```diff
@@ -9,7 +9,7 @@
     <!-- Read document collection-->
     <mapping>
       <target>rest_documents_GET@libfty_security_wallet_rest</target>
-      <url>^/api/v1/admin/security-wallet/portfolios/([a-zA-Z-]+)/documents$</url>
+      <url>^/api/v1/admin/security-wallet/portfolios/([a-zA-Z-]+)/documents.*$</url>
```

Comment:
Are you sure about the need to change the regex ? If it's needed to add the parameters (documents?usage=usage) we need to make the regex more explicit but I'm not sure it's needed as we could use <args></args>

Reply:
It won't match the REST API call without changing the regex, but we can make it more explicit

Reply:
so we could have have something like documents($|?.+$)

Reply:
Fixed